SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -378,6 +378,9 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Cloud", meta = (DisplayName = "Deployment tags"))
 	FString DeploymentTags;
 
+	UPROPERTY(EditAnywhere, config, Category = "Cloud", meta = (DisplayName = "Auto generate configuration from current map"))
```

Comment:
I believe the variables in this section have been hidden from the settings window (all `UPROPERTY` tags removed except `config`) [in the RC](https://github.com/spatialos/UnrealGDK/blob/0.10.0-rc/SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h#L378), so I suggest this one is hidden as well.
```suggestion
	UPROPERTY(config)
```